prereise/gather/flexibilitydata/create_cache_files.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from doe import doe_data
+from geo import geo_data
+
+
+def collect_all_raw_data(relative_download_path = "raw"):
+    """download all required raw data needed for producing cached files
+
+    param string download_path: folder to store the downloaded file
+    :return: (*None*)
+    """
+    
+    file_dir = os.path.abspath(os.path.dirname(__file__))
+    download_path = os.path.join(file_dir, relative_download_path)
+    
+    geo_data.get_census_data(download_path)
+    geo_data.get_crosswalk_data(download_path)
+    geo_data.get_LSE_region_data(download_path)
+    geo_data.get_county_fips_data(download_path)
+
+def create_geo_cache_files(relative_raw_path = "raw", relative_cache_path = "cache"):
+    """process downloaded raw files and create cached intermediate files
+
+    param string raw_data_path: folder that contains downloaded raw data
+    param string cache_path: folder to store processed cache files
+    :return: (*None*)
+    """
+    
+    file_dir = os.path.abspath(os.path.dirname(__file__))
+    raw_path = os.path.join(file_dir, relative_raw_path)
```

Comment:
The folder is created first in the only function that calles these things, and the files must be created first before reaching here. I can add checks here as well if there will be some occasion when these functions are used stand-alone.

Reply:
Yeah, it is my understanding that we generally try to include error-checking functionality in case these functions are used individually. I can definitely see some of these functions being used individually if a user wants to borrow some of the data formatting/cleaning functionality for their own purposes. That being said, @BainanXia or @danielolsen could probably offer better guidance on how much (if any) error checking is actually needed.